src/Bookings.js

```diff
@@ -1,19 +1,77 @@
 import React, { Component } from "react";
 import Search from "./Search.js";
-// import SearchResults from './SearchResults.js';
-// import FakeBookings from '../data/fakeBookings.json';
+import SearchResults from "./SearchResults.js";
 
 export default class Bookings extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      FakeBookings: null,
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+    fetch("https://cyf-react.glitch.me")
+      .then(res => {
+        if (res.status >= 200 && res.status < 300) {
+          return res;
+        } else {
+          throw new Error("HTTP error");
+        }
+      })
+      .then(res => res.json())
+      .then(data =>
+        this.setState({
+          FakeBookings: data,
+          isLoading: false,
+          SearchResults: data
+        })
+      )
+      .catch(err => {
+        this.setState({
+          err: err
+        });
+      });
+  }
+
+  // Match customer.firstName or customer.surname
   search = searchVal => {
-    console.info("TO DO!", searchVal);
+    let filteredSearch = this.state.FakeBookings.filter(customer => {
+      return searchVal === customer.surname || searchVal === customer.firstName;
+    });
+
+    if (filteredSearch.length === 0) {
+      this.setState({ SearchResults: this.state.FakeBookings });
+    } else {
+      this.setState({ SearchResults: filteredSearch });
+    }
+  };
+
+  loadingMessage = () => {
+    return <span>Loading...</span>;
+  };
+
+  errorMessage = () => {
```

Comment:
We haven't talked about it much it class but for this case I would prefer to put this in it's own little component. It shows up nicer in then dev tools then.

Usually if I'm making something like this where the component will only be shown in this one file, that's where I break the rule about how every component should be in it's own file.